### PR TITLE
chore: release 1.1.0

### PR DESCRIPTION
**Pull request by znorgaard**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/fastquorum/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/fastquorum _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md
Comment (SPPearce) [inline CHANGELOG.md:1]: Need to mention updating the tool versions for the nf-core modules.

Comment (znorgaard) [reply CHANGELOG.md:1]: How's this update?
https://github.com/nf-core/fastquorum/pull/90/commits/9d93869512fc91f6bb94d2a71376012eadeeff43
Included on #90

File: CHANGELOG.md
Comment (SPPearce) [inline CHANGELOG.md:6]:
```suggestion
## [[1.0.2]](https://github.com/nf-core/fastquorum/releases/tag/1.0.2) -- 2024-11-16
```
Or whenever it actually gets released!

**Review by SPPearce — APPROVED** [review]
Looks good to me!

**Review by SPPearce — APPROVED** [review]
Looks good to me, apart from the fact that the download test is failing 😞

**Comment by SPPearce** [comment]
Although I see @MatthiasZepper is already involved in looking at that.

**Comment by MatthiasZepper** [comment]
> Although I see @MatthiasZepper is already involved in looking at that.

Yes, the botched Seqera Containers introduction is keeping me up on my toes. But the error here was nicely troubleshooted, pinpointed and reported by @znorgaard, so I could fix it. The download will work, once [this PR is merged to tools](https://github.com/nf-core/tools/pull/3293) and already does, [when the action from the dev branch is used instead the version from master](https://github.com/nf-core/fastquorum/actions/runs/11984842471). 

I do really not know how to proceed, if one doesn't want to wait for the tools update. I think, in this case, it would be fine to merge with a failing test. Otherwise, it is a hen and an egg problem: To merge `dev` to `master`, you need to pass the test version on `master`, that will be corrected with the merge from `dev`.

**Comment by SPPearce** [comment]
Happy to merge with the failed test then, if it'll be fixed anyway. Although that'll require someone with more permissions than me.

**Comment by MatthiasZepper** [comment]
@znorgaard: `nf-core download` fix was reviewed and merged earlier today, so the test is now passing. Currently, on `dev` (_sufficient for making the test pass_), but there is a 3.1 release of nf-core tools expected some when later this week, so ordinary people will also be able to successfully download the pipeline. 

Happy fastquorum releasing!

File: workflows/fastquorum.nf
Comment (clintval) [inline workflows/fastquorum.nf:224]: The syntax formatting here makes this an especially difficult section to read/review.

Could we format this?

File: CHANGELOG.md
Comment (clintval) [inline CHANGELOG.md:28]:
```suggestion
- [PR #87](https://github.com/nf-core/fastquorum/pull/87) - Raise minimum Nextflow version to 24.04.2
```

**Review by clintval — APPROVED** [review]

File: workflows/fastquorum.nf
Comment (znorgaard) [reply workflows/fastquorum.nf:224]: https://github.com/nf-core/fastquorum/issues/97

**Pull request merged**
